packages/zoe/test/unitTests/contracts/test-automaticRefund.js

```diff
@@ -122,34 +116,29 @@ test('zoe - automaticRefund same issuer', async t => {
 test('zoe with automaticRefund', async t => {
   try {
     // Setup zoe and mints
-    const {
-      issuers: defaultIssuers,
-      mints,
-      amountMaths,
-      moola,
-      simoleans,
-    } = setup();
-    const issuers = defaultIssuers.slice(0, 2);
+    const { issuers, mints, moola, simoleans } = setup();
+    const [moolaIssuer, simoleanIssuer] = issuers;
+    const [moolaMint, simoleanMint] = mints;
```

Comment:
This is an improvement. I like the style of calling them moolMint rather than mints[0]